dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -269,6 +272,7 @@ def test_profile(self):
         expected_profile = dict(
             min=2.0,
             max=6.0,
+            sum=8,
```

Comment:
How did you get 8 here? Because the sum of the numbers is 20.5, and after they get converted to strings there seems like there should be more than 8 characters

Reply:
Values which are not integers do not get included